src/bp-forums/forums/template.php

```diff
@@ -884,6 +884,9 @@ function bbp_list_forums( $args = '' ) {
 				$counts = $r['count_before'] . implode( $r['count_sep'], $count ) . $r['count_after'];
 			}
 
+			// Change the style of link tag.
+			$r['link_before'] = apply_filters( 'bb_forum_link_custom_attr', $r['link_before'], $title );
```

Comment:
@kishanjasani This filter should be for the entire `$r` not only for link_before. 

```$r = apply_filters( 'bb_forum_link_custom_attr', $r, $title );```

Reply:
Thanks for the review. Did changes as per suggestion.